### PR TITLE
Add umip-11 for renBTC collateral

### PR DESCRIPTION
**Pull request by adrianmcli**
See original Google Doc draft here: https://docs.google.com/document/d/1lBpfVVdkVB2SmGQVQwjUCVu7hU7IvnyaLMcadLW8YZg/edit#

### CODE REVIEW

File: UMIPs/umip-11.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- A final fee of 0.018 needs to be added for renBTC in the Store contract.
+- The precision for the USDBTC price identifier needs to be amended to 8 decimals instead of the usual 18 in order to comply with renBTC’s 8 decimals of precision.
+
+## Rationale
+
+The rationale behind this change is giving deployers more useful collateral currency options. renBTC needs to be used instead of BTC due to it being an ERC20 representing BTC on the Ethereum blockchain, whereas native BTC does not exist on the Ethereum blockchain.
+
+renBTC was chosen in particular due to the fact that renBTC is the most popular (and largest) truly decentralized BTC-representing coin on the Ethereum blockchain. It is not being chosen at the exclusion of other possible BTC-representing coins, but it is a natural first choice for this purpose.
+
+While wBTC currently has larger market dominance than renBTC, it is worth noting that renBTC has already gained significant traction in the DeFi space. There exists a large amount of renBTC liquidity on Curve Finance ($98m), Balancer ($5m), and Uniswap (\$150k).
```
Comment (nicholaspai) [inline UMIPs/umip-11.md:36]: What's with the "\" before 150k?

File: UMIPs/umip-11.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+Large projects like Compound and Maker already have wBTC as a collateral type, so using renBTC within the UMA ecosystem can serve to differentiate the UMA community in terms of promoting a more decentralized version of BTC on the Ethereum blockchain.
+
+Recently, Yearn Finance governance has also voted overwhelmingly in favor of renBTC against its competitors.
```
Comment (nicholaspai) [inline UMIPs/umip-11.md:40]: I actually don't think this is accurate about yearn anymore, the vote is actually strongly against adding renBTC as collateral due its current centralized custody.

Screen Shot 2020-08-28 at 12 43 09

[YIP](https://yips.yearn.finance/YIPS/yip-42)

File: UMIPs/umip-11.md (hunk 1 above)
Comment (adrianmcli) [reply UMIPs/umip-11.md:36]: ah shit, it's Prettier being annoying: https://github.com/prettier/prettier/issues/6213

I'll fix it manually.

File: UMIPs/umip-11.md (hunk 2 above)
Comment (adrianmcli) [reply UMIPs/umip-11.md:40]: oh wow, interesting. I'll just remove this sentence then.

**Pull request merged**